cookbook/wandb_logged_chain/wandb_chain.py

```diff
@@ -34,21 +32,28 @@ class Settings(BaseSettings):
         start_time_ms=round(datetime.datetime.now().timestamp() * 1000),
         metadata={"user": "mirascope_user"},
     )
-    chat = OpenAIChat(api_key=settings.openai_api_key)
 
-    who_prompt = WhoPrompt(span_type="tool", person="Brian")
-    who_completion = chat.extract(Person, who_prompt)
-    who_span = who_prompt.trace(who_completion, root_span)
+    who_prompt = WhoPrompt(
+        span_type="tool",
+        person="Brian",
+        api_key=settings.openai_api_key,
```

Comment:
you should just run `os.environ["OPENAI_API_KEY"] = settings.openai_api_key` at the top. Same for the remaining prompts.